### ISSUE TITLE
Allow restarting numeration from a specific number

### ISSUE THREAD

**Issue by elad2109** [flameshot-org/flameshot#1214]
Sometimes I want to numerate a few series on the same screenshot.
I would love to be able to start a new numeration with "1" again.

Also sometimes I have a saved screenshot with numeration on it.
I would love to add a new number starting from the last number already appears.

**Comment by mmahmoudian** [comment]
For continuing a number series, this is what I do at the moment: click outside of the capture area enough times to get to the number I want and then I continue as usual.

**Comment by elad2109** [comment]
Thanks, would still love to have a built-in feature.

**Comment by borgmanJeremy** [comment]
Now that we have customizable hot keys this is actually much easier to add. We can bind it to dedicated keys.

**Comment by elad2109** [comment]
> Now that we have customizable hot keys this is actually much easier to add. We can bind it to dedicated keys.


You mean "We can bind it" - as the developers or the users?

**Comment by borgmanJeremy** [comment]
Sorry for the confusion, I mean as in the developers.

**Comment by Creteil** [comment]
Actually, using the « Auto incrementing counter bubble », it is impossible to start at a manually chose number when using it.

Why not assigning the « CTRL + Scroll wheel » to set the starting number of the bubble ?

This actually already show numbers increasing/decreasing, just add the logic :

When used with « CTRL + Scroll wheel », **just set the bubble number to the one show in upper left and don't change size of the bubble**...

Look like there not much to change since the big part of the code already exist :

https://user-images.githubusercontent.com/27310214/111832251-757f4f80-88e8-11eb-9727-d8986abb9a8b.mp4

**Comment by borgmanJeremy** [comment]
Yes I would like to add this now. I was mostly waiting for custom shortcuts to be merged. Now we can add this feature and let users customize how to trigger it.

**Comment by Drarig29** [comment]
Are you still working on this @borgmanJeremy?

**Comment by Drarig29** [comment]
I see you are working on other things so I made a PR.

**Comment by Drarig29** [comment]
> Yes I would like to add this now. I was mostly waiting for custom shortcuts to be merged. Now we can add this feature and let users customize how to trigger it.

This custom shortcut is still a good idea though!

And I think it is easier to implement with my PR (#2162) because it would only consist in resetting `m_context.circleCount` (and show the notifier box to have a visual feedback).

**Comment by aquatic7** [comment]
What is the status on this request after almost 3 years?
It would be so useful to be able to edit the number like you're able to in fx Snagit.

**Comment by milnadar** [comment]
> What is the status on this request after almost 3 years? It would be so useful to be able to edit the number like you're able to in fx Snagit.

Is there a video of how this similar feature works in Snagit?

**Comment by serebryakov02** [comment]
Hi, I took a look at this and it still seems like a useful feature request.

A simple user-facing behavior could be to let users change the next number used by the numbered-circle tool, for example by increasing, decreasing, or resetting it before placing the next circle.

That would make it possible to start a new sequence at `1`, or continue numbering from a specific value on an existing screenshot, without changing any numbered circles that were already placed.

Would it be okay if I work on this and open a PR?

**Comment by borgmanJeremy** [comment]
This actually is completed via #4698

**Comment by serebryakov02** [comment]
@borgmanJeremy Sorry, I missed #4698 when I looked at this.

Thank you for pointing it out. If #4698 covers this request, then there is no need for me to open a separate PR or follow up further here.
